code/modules/mob/living/carbon/human/species/xenomorphs/alien_powers.dm

```diff
@@ -124,6 +124,29 @@
 		new /obj/effect/alien/weeds/node(loc)
 	return
 
+/mob/living/carbon/human/proc/Spit(var/atom/A)
+	if((last_spit + 1 SECONDS) > world.time) //To prevent YATATATATATAT spitting.
+		to_chat(src, "<span class='warning'>You have not yet prepared your chemical glands. You must wait before spitting again.</span>")
+		return
+	else
+		last_spit = world.time
+
+	if(spitting && (stat || paralysis || stunned || weakened || lying || restrained() || buckled))
```

Comment:
You might be able to use ``if(spitting && !incapacitated(INCAPACITATION_DISABLED) )`` here instead